src/selectors/metadata.js

```diff
@@ -24,6 +23,14 @@ const runCommandTemplates = {
   task: (name) => `kedro run --to-nodes ${name}`,
 };
 
+const renderParameters = (params) => {
+  if (!params || !params[Object.keys(params)[0]]) {
```

Comment:
Sorry what does `Object.keys(params)[0]]` mean? Also if we are going to reuse it, can we extract it into a variable and give it a descriptive name?

Reply:
i have edited this code so it's more clear. changed renderParameters to unwrapParameters. it's actually because of the way we get the json from backend. it looks like this 
{ 
   params_name: {
       params_key:1 { 
       
      }
} --> there's this extra wrapper around the json, and I need to unwrap it and access the first object to get all the parameters output.